plugins/aws/iam/rootAccountInUse.js

```diff
@@ -27,6 +27,10 @@ module.exports = {
 	},
 
 	run: function(cache, settings, callback) {
+		this._run(cache, settings, callback, new Date())
+	},
+
+	_run: function(cache, settings, callback, now) {
```

Comment:
Is there a benefit to doing this vs just:
```
run: function(cache, settings, callback) {
    var date = new Date();
    .... // rest of the code as-is
},
```

Reply:
Yes - testability (as below). I wanted to have tests that inject different values for now so that I could check different conditions.

Reply:
You can see this in use with `rootAccountInUse.spec.js` which supplies a number of different dates for `now` and then evaluates what the rule says.